BpRobotics/Client/bprobotics-client/src/components/AddCustomer.js

```diff
@@ -0,0 +1,202 @@
+import React from 'react';
+import { useNavigate } from "react-router-dom";
+import { useState } from "react";
+import LoadingSpin from "react-loading-spin";
+import '../App.css';
+import Button from 'react-bootstrap/Button';
+import Form from 'react-bootstrap/Form';
+import useAxios from '../hooks/useAxios';
+import useAxiosFetchGet from '../hooks/useAxiosFetchGet';
+
+
+function AddCustomer() {
+    const axiosInstance = useAxios();
+    const url = `${process.env.REACT_APP_HOST_URL}/api/customers`;
```

Comment:
You don't need `${process.env.REACT_APP_HOST_URL}`, it is already defined in axiosInstance